client/src/pages/DashboardPage.vue

```diff
@@ -105,11 +111,18 @@
 import AvatarImage from "src/components/atoms/AvatarImage.vue"
 import { useCurrentUser } from "src/use/user"
 import { useQuery } from "@vue/apollo-composable"
-import { CURRENT_USER_SUBMISSIONS } from "src/graphql/queries"
+import { CURRENT_USER_SUBMISSIONS, GET_SUBMISSIONS } from "src/graphql/queries"
 import SubmissionTable from "src/components/SubmissionTable.vue"
 import { computed } from "vue"
 
 const { currentUser } = useCurrentUser()
+const { result: all_submissions_result } = useQuery(GET_SUBMISSIONS, {page: 1})
+const all_submissions = computed(() => {
+  let s = all_submissions_result.value?.submissions.data ?? []
+  return [...s].sort((a, b) => {
+    return new Date(b.created_at) - new Date(a.created_at)
+  })
+})
```

Comment:
This needs to be sorted on the backend side rather than the client.  Because the query is paginated, this will only sort the /page/ of results.

Reply:
Refactored as suggested.